docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
+
+.Procedure
+
+. Resolve issues detected by the {ProductShortName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), right-click the issue and select *Preview Quick Fix*. To accept the suggested fix, right-click the issue and select *Apply Quick Fix.*
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code* to open the file involved, make any changes needed to the code, and then save the file.
+
+. Optional: After you fix an issue, you can right-click it and select *Mark as Complete*. The extension displays the Resolved icon (image:vs_resolved.png[Resolved]) next to the issue. This has no effect on the code.
+
+. Optional: To remove an issue from the list, right-click it and select *Delete*. This has no effect on the code.
```

Comment:
I'm not sure you need to document "Delete". It's pretty obvious. I also do not think you need to say "x has no effect on the code". All these actions are performed on the issues in MTA, not on the code itself. If you think this merits a general sentence in the intro, fine. But you do not need it for each issue option.